app/routines/search_users.rb

```diff
@@ -39,32 +43,37 @@ class SearchUsers
   def exec(query, options={})
```

Comment:
Since you won't be using `SearchUsers` in the endpoint controller method, we can leave this code out for now (maybe we'll pull it in at some point, but maybe not for a while.